### PR TITLE
Add ability to skip summary output. Also support a LEFT_QUIET env variable

### PR DESCRIPTION
**Pull request by washtubs**
Decided to tackle some further issues raised [here](https://github.com/evilmartians/lefthook/issues/78#issuecomment-658333189). There was mention of using a global environment variable so you didn't need to rely on the yaml having it configured. This is personal preference after all.

*My* personal preference is I like it very quiet. So I made it so you could get rid of the summary line as well.

### CODE REVIEW

File: docs/full_guide.md [hunk 1]
```diff
@@ -469,6 +469,28 @@ source_dir: ".lefthook"
 source_dir_local: ".lefthook-local"
 ```
 
+## Mange verbosity
```
Comment (Envek) [inline docs/full_guide.md:472]: Typo:
```suggestion
## Manage verbosity
```

**Review by Envek — COMMENTED** [review]
Looking good to me, except one typo.

Also it would be awesome to have 2 commits: one with option for summary output suppresion and one with environment variable addition.

**Comment by washtubs** [comment]
Done and done!

**Review by Envek — APPROVED** [review]

**Comment by Envek** [comment]
Awesome, thank you!

**Pull request merged**